### PR TITLE
Deserialization of Untrusted Data

### PR DESCRIPTION
**Pull request by bhaskarvilles**
introduced through: unknown:unknown@0.0.0 › com.google.code.gson:gson@2.8.6
Fix: Upgrade to com.google.code.gson:gson@2.8.9
Affected versions of this package are vulnerable to Deserialization of Untrusted Data via the writeReplace() method in internal classes, which may lead to DoS attacks.

### CODE REVIEW

File: GhidraBuild/BuildFiles/JsonDoclet/build.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ eclipse.project.name = '_JsonDoclet'
 apply plugin: 'java-library'
 
 dependencies {
-	api "com.google.code.gson:gson:2.8.6"
+	api "com.google.code.gson:gson@2.8.9"
```
Comment (ryanmkurtz) [inline GhidraBuild/BuildFiles/JsonDoclet/build.gradle:22]: We use the `:` style everywhere.  Would you mind sticking with that convention?

**Review by ryanmkurtz — CHANGES_REQUESTED** [review]
Thanks for letting us know about this.  The JsonDoclet uses gson only at build time.  While it's important to fix that, the more important one is in `Framework/Generic/build.gradle`, which includes the 2.8.6 version we ship in the Ghidra distribution.  Would you mind fixing that one too?

**Comment by bhaskarvilles** [comment]
Sure,Will definitely try to work & make a new version.

File: GhidraBuild/BuildFiles/JsonDoclet/build.gradle (hunk 1 above)
Comment (bhaskarvilles) [reply GhidraBuild/BuildFiles/JsonDoclet/build.gradle:22]: Ok

**Comment by mumbel** [comment]
@bhaskarvilles   looks like you might have not pushed your latest change (unless the request for review was accidental)

**Review by bhaskarvilles — COMMENTED** [review]
Changed..

**Review by ryanmkurtz — APPROVED** [review]
Thanks.  I'll go ahead and fix up `Framework/Generic/build.gradle` and get this merged in.

**Comment by bhaskarvilles** [comment]
Thanks 😊.

**Comment by ryanmkurtz** [comment]
Merged in by c6d89deacf5cd65f8df1a5ef19da96ce0c3ffe7c.  Backported to `patch`.

**Pull request closed without merging**